examples/starfield/main.rs

```diff
@@ -34,14 +35,13 @@ impl Starfield {
         let h = framebuffer.var_screen_info.yres as usize;
 
         for star in self.stars.iter_mut() {
-            let star_data = star.tick(w, h);
-            if star_data.0 < w && star_data.1 < h {
-                Starfield::draw_star(star_data, framebuffer, frame);
-            }
+            Starfield::draw_star(&star, framebuffer, frame, (0, 0, 0));
+            star.tick(w, h);
+            Starfield::draw_star(&star, framebuffer, frame, star.color);
```

Comment:
Nice optimization!

Reply:
Thanks! I'm running this on a super low power device, so the difference was really noticeable.